integration-tests/tests/run-catalog-workflow.js

```diff
@@ -1,28 +1,26 @@
 const _ = require('lodash/fp')
-const { checkbox, click, clickable, clickTableCell, input, waitForNoSpinners } = require('../utils/integration-utils')
-const { enableDataCatalog, withWorkspace } = require('../utils/integration-helpers')
+const { checkbox, click, clickable, clickTableCell, input, noSpinnersAfter, waitForNoSpinners } = require('../utils/integration-utils')
+const { checkBucketAccess, enableDataCatalog, withWorkspace } = require('../utils/integration-helpers')
 const { withUserToken } = require('../utils/terra-sa-utils')
 
 
 const testCatalogFlowFn = _.flow(
   withWorkspace,
   withUserToken
-)(async ({ page, testUrl, token, workspaceName }) => {
+)(async ({ page, testUrl, token, workspaceName, billingProject }) => {
   await enableDataCatalog(page, testUrl, token)
   await click(page, clickable({ textContains: 'browse & explore' }))
-  await waitForNoSpinners(page)
 
   await click(page, checkbox({ text: 'Granted', isDescendant: true }))
   await clickTableCell(page, 'dataset list', 2, 2)
-  await waitForNoSpinners(page)
-  await click(page, clickable({ textContains: 'Link to a workspace' }))
-  await waitForNoSpinners(page)
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
 
   await click(page, clickable({ textContains: 'Start with an existing workspace' }))
   await click(page, input({ labelContains: 'Select a workspace' }))
   await click(page, `//*[@role="combobox"][contains(normalize-space(.), "${workspaceName}")]`)
   await click(page, clickable({ textContains: 'Import' }))
   await waitForNoSpinners(page)
+  await checkBucketAccess(page, billingProject, workspaceName)
```

Comment:
It's definitely not useful here because the "Import" isn't actually getting clicked on (that's what I see when I run the test on dev locally with Chrome visible). The page URL check won't actually fail the test.

See [my PR](https://github.com/DataBiosphere/terra-ui/pull/2811/files#r803768256) where I'm commenting out these lines because the clickable xpath being used for "Import" is changing in my PR to require the button to be enabled. It then causes this test to fail because Import is not actually enabled (and thus the click is a no-op).